celery_dyrygent/workflows/node.py

```diff
@@ -46,12 +47,17 @@ def to_dict(self):
         """
         Packs internals into a serializable dict
         """
-        return dict(
+        d = dict(
             id=self.id,
             # celery signatures are serializable so it will work
             signature=self.signature,
             dependencies=self.dependencies,
         )
+        if not self.user_params :
```

Comment:
I don't think there is a need for this ```if```.
Let's always return dict with ```user_params``` or ```custom_payload``` however we decide to call it.

Reply:
Fine. I only wanted to make sure that object that were serialized with versions up to 0.7.0 could still be deserialized, but i'll modify those checks as you suggested.

Reply:
No worries. It works. I just tested with wf published using 0.6.1 and processed with 0.7.0 + your changes. Works flawlessly